rclcpp_components/cmake/rclcpp_components_package_hook.cmake

```diff
@@ -13,6 +13,11 @@
 # limitations under the License.
 
 # register node plugins
-ament_index_register_resource(
-  "rclcpp_components" CONTENT "${_RCLCPP_COMPONENTS__NODES}")
+list(REMOVE_DUPLICATES _RCLCPP_COMPONENTS_PACKAGE_RESOURCE_INDICES)
+foreach(resource_index ${_RCLCPP_COMPONENTS_PACKAGE_RESOURCE_INDICES})
+  #message(WARNING "registered index: ${resource_index}")
+  #message(WARNING "content for index: ${_RCLCPP_COMPONENTS_${resource_index}__NODES}")
```

Comment:
I'll remove these lines before merging. I let them in for now in case the interested reviewers wants to debug ;-)

Reply:
Reminder to remove before merging.